src/lesson1/HomeWork/HomeWork.java

```diff
@@ -0,0 +1,71 @@
+package lesson1.HomeWork;
+
+
+public class HomeWork {
+
+    // Написать метод вычисляющий выражение a * (b + (c / d))
+    // и возвращающий результат с плавающей точкой,
+    // где a, b, c, d – целочисленные входные параметры этого метода;
+    private static float calculating(int a, int b, int c, int d){
+        return a * (b+((c*1.0f) / d));
+    }
+
+    // Написать метод, принимающий на вход два целых числа,
+    // и проверяющий что их сумма лежит в пределах от 10 до 20(включительно),
+    // если да – вернуть true, в противном случае – false;
+    private static boolean checkSum(int a, int b){
+
+        return (a+b)<=20 && (a+b)>=10;
+    }
+
+    // Написать метод, которому в качестве параметра передается целое число,
+    // метод должен проверить положительное ли число передали, или отрицательное.
+    // Замечание: ноль считаем положительным числом. Результат работы метода вывести в консоль
+    private static String checkNumber(int a){
+        String result;
+        if (a < 0)
+            result = String.format("Число %d отрицательное", a);
+        else
+            result = String.format("Число %d положительное", a);
+        return result;
+    }
+
+    // Написать метод, которому в качестве параметра передается строка, обозначающая имя,
+    // метод должен вернуть приветственное сообщение «Привет, переданное_имя!»;
+    // Вывести приветствие в консоль.
+    private static String helloUser(String name){
+        return String.format("Привет, %s!", name);
+    }
+
+    // *Написать метод, который определяет является ли год високосным.
+    // Каждый 4-й год является високосным, кроме каждого 100-го,
+    // при этом каждый 400-й – високосный.
+    // Для проверки работы вывести результаты работы метода в консоль
+    private static String checkLeapYear(int year){
+        String result;
+        if ( (year % 400 == 0) || ( (year % 4 == 0) && (year % 100 != 0) ) )
+            result = String.format("Год %s - високосный", year);
+        else result = String.format("Год %s - не високосный", year);
+        return result;
+    }
```

Comment:
Кажется тут лучше boolean. И результат тоже надо для проверки в консоль. Вообще можно в одну строку решить, если не выводить, например:
```suggestion
    private static boolean checkLeapYear(int year) {
        return (year % 400 == 0) || (year % 4 == 0) && (year % 100 != 0)
    }
```